frontend/src/modules/editor/global/less/editor.less

```diff
@@ -100,35 +100,80 @@
 }
 
 .editor-item-settings {
-    position: absolute;
-    top: 0%;
-    left: 0px;
     font-size: 16px;
     width: 100%;
     opacity: 1;
     background-color: transparent;
     transition: all 0.3s;
 
+    
     a {
       text-decoration: none;
     }
-
+    
     &:hover {
       cursor: move;
     }
-
+    
     &-inner {
-      padding: 6px;
-    }
+      button {
+        padding: 10px;
+        font-size: 16px;
+        line-height: 16px;
+        height: 16px;
+        box-sizing: content-box;
+        text-align: center;
+      }
 
-    .editor-delete-page-element {
-      float: right;
+      width: 100%;
+      box-sizing: border-box;
+      display: -ms-flexbox;
+      display: -webkit-flex;
```

Comment:
Prefixed attributes not required in modern browsers.

Reply:
but we can leave them in ... as it will help support older versions and not break modern browsers

Reply:
I'd say we should probably try to avoid adding anything specifically to support legacy browsers (particularly as the framework no longer supports pre-v11 IE).